### DISCUSSION TITLE
Unable to verify accademic affiliation - document in French.

### DISCUSSION THREAD

**Discussion by giuscost** [community/community#121676] [GitHub Education]
### Hi everyone,

Hello,

I'm new to GitHub and I'm trying to correctly setup my account. In order to join the academic community, I have to prove my academic affiliation. Unfortunately, my university (HES-SO, Switzerland) does not issue badges with pictures for the academic personnel.

Therefore I have at hand a certificate provided by HR that states my status as associate professor. Moreover, I do have a personal webpage within the school domain, on top of the institutional email.

Upon uploading the .png version of the affiliation letter, which is written in French, the system does not recognize the relevant information to move forward with my application.

Could anyone of the support team help me?

Thanks in advance.
Kind regards.
Giuseppe Costanzo.

**Comment by samus-aran** [comment]
Hi @giuscost 

> Upon uploading the .png version of the affiliation letter, which is written in French, the system does not recognize the relevant information to move forward with my application.

Reviewing the application, I don't believe that is what was uploaded 😅 Try again and ensure that you can preview the live photo before submitting.
